src/main/scala/com/horizon/exchangeapi/BusinessRoutes.scala

```diff
@@ -682,6 +706,17 @@ trait BusinessRoutes extends JacksonSupport with AuthenticationSupport {
       ]
     }
   ],
+    "secretBinding": [
+     {
+       "serviceOrgid": "string",
+        "serviceUrl": "string",
+        "serviceArch": "amd64",
+        "serviceVersionRange": "x.y.z",
+         "secrets": [
```

Comment:
Make sure `"secrets": ...` is structurally similar to `"inputs": ...` above, but with the context of storing secrets instead of inputs.